h/tasks/queue.py

```diff
@@ -0,0 +1,27 @@
+# pylint: disable=no-member # Instance of 'Celery' has no 'request' member
+
+
+from h.celery import celery, get_task_logger
+
+log = get_task_logger(__name__)
+
+
+@celery.task
```

Comment:
Task just about queueing here.

Reply:
Yes, my idea is to add that in a follow up, it will be easier to diff without this refactor out of the way.